Projekty/Projekt1/Adrian_Stańdo__Mateusz_Stączek__Paweł_Wojciechowski/feature_engineering.ipynb

```diff
@@ -0,0 +1,1966 @@
+{
```

Comment:
<p>Nie sprawdziliście liczności tych klas.</p><p> </p><p>Trzeba zdropować jedną kolumnę, bo mamy pełną zależność (jeśli nie less, nie more, to wiadomo, że USA)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>